### ISSUE TITLE
And the updates?

### ISSUE THREAD

**Issue by marcellorg** [formvalidation/support#887]
Has this plugin been discontinued? I'm seeing that for some time it has no update, it's still in beta bootstrap.

**Comment by jacobbroberg** [comment]
@nghuuphuoc please give us a notice if you have discontinued this! People have paid for this component, at the very least they deserve to know!

**Comment by Eurymachus** [comment]
I think the way to combat this is to make the source code public.
We all have it, we could fix any issues as part of a community.

**Comment by jacobbroberg** [comment]
It used to be public until version 0.7.0, when @nghuuphuoc started charging a fee - which to me is acceptable if he intends to maintain it. If not, then make it public again like any other open source component where the originator in principle has no other obligation than to give people a chance to fix bugs themselves.

**Comment by Eurymachus** [comment]
Agreed. Is there not some sort of license clause that says we can make the source public ourselves?

**Comment by jacobbroberg** [comment]
Email sent to @nghuuphuoc at addresses: phuoc@huuphuoc.me and hello@formvalidation.io

To Phuoc Nguyen

Our company bought a license to Formvalidation.io when it moved from an open-source project to a commercial project. We have so far been very happy to use the component.

But it is discouraging to see that you have apparently abandoned the project - no updates since 2016!

I assume that you have seen the discussion in Github:
https://github.com/formvalidation/support/issues/887
https://github.com/formvalidation/support/issues/876

If you for some reason do not wish to continue developing and supporting the component, you should have the decency to tell people (e.g. here: http://formvalidation.io/news/) and consider allowing others to continue fixing bugs and develop by making the code public available again.

**Comment by manovraresoft** [comment]
I think he died.

**Comment by manovraresoft** [comment]
There was rumor about him dying a while back.  He had gotten married, and he was still maintaining and then I believe shortly after he had died. 

He has no updates to any social media or anything around the same time support has dropped off.  Begin looking for a new solution guys.

**Comment by jacobbroberg** [comment]
That would definitely be an understandable (and tragic) explanation! 
His personal twitter was updated in October 2017: https://twitter.com/nghuuphuoc
And apparently someone is committing to his GitHub account in the last couple of months (https://github.com/nghuuphuoc) - unless I misunderstand GitHubs statistics: 
![image](https://user-images.githubusercontent.com/8348616/38929558-05cc82a8-430d-11e8-950d-e9d75da9d7d0.png)

But your probably right - no matter the explanation, we probably need to find another solution.

**Comment by manovraresoft** [comment]
I seen that on GitHub reports as well.  I remain hopeful. He is a brilliant young guy.

I am checking out [https://jqueryvalidation.org/](https://jqueryvalidation.org/). It seems like an easy to use and customize-able html + jquery solution.

With Bootstrap 4 out, I got to start migrating my form validation to work.  The validation tool above will prove its worth if I can write stuff for Bootstrap 4 changes.
